New Broken Links (#2397)
MicrosoftDocs/edge-developer: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

| Approach | Pros and Cons | Article |
|---|---|---|

> | Browser > DevTools > **Elements** tool | You have to manually copy changes from DevTools into your source files. | [Inspect, edit, and debug HTML and CSS with the Elements tool](./microsoft-devtools-guide  /microsoft-edge/devtools-guide-chromium/elements-tool/elements-tool) |

Reviewer: ```suggestion
| Browser > DevTools > **Elements** tool | You have to manually copy changes from DevTools into your source files. | [Inspect, edit, and debug HTML and CSS with the Elements tool](./microsoft-devtools-guide/microsoft-edge/devtools-guide-chromium/elements-tool/elements-tool) |
```
test after removing spaces

> | Browser > DevTools > **Sources** tool > **Filesystem** tab to define a **Workspace** | Always saves the changes, which could discourage experimenting | [Using the Filesystem tab to define a local Workspace](/microsoft-edge/devtools-guide-chromium/sources/index) in _Sources tool overview_ |

Reviewer: ```suggestion
| Browser > DevTools > **Sources** tool > **Filesystem** tab to define a **Workspace** | Always saves the changes, which could discourage experimenting | [Using the Filesystem tab to define a local Workspace](../devtools-guide-chromium/sources/#using-the-filesystem-tab-to-define-a-local-workspace) in _Sources tool overview_ |
```
try to test the link, since relative ../

> | Browser > DevTools > **Settings** > **Experiments** > **Open source files in Visual Studio Code** | Always saves the changes, which could discourage experimenting | Experiments > [Opening source files in Visual Studio Code](/microsoft-edge/devtools-guide-chromium/sources/opening-sources-in-vscode). |

Reviewer: ```suggestion
| Browser > DevTools > **Settings** > **Experiments** > **Open source files in Visual Studio Code** | Always saves the changes, which could discourage experimenting | Experiments > [Opening source files in Visual Studio Code](../devtools-guide-chromium/sources/opening-sources-in-vscode.md). |
```
try to test the link, since relative ../
